client/src/Components/QuestionsAndAnswers/QuestionWithAnswers/IndividualAnswer.jsx

```diff
@@ -1,6 +1,7 @@
 import React, {useState} from 'react';
 import { format, parseISO } from 'date-fns';
-import styled from 'styled-components'
+import styled from 'styled-components';
+var axios = require('axios')
```

Comment:
Might want a semicolon here after `require('axios')` though as it is right now, it shouldn't cause any problems